app/javascript/components/Product.jsx

```diff
@@ -226,3 +230,42 @@ const CommentInfo = ({ product }) => {
     </>
   )
 }
+
+const TrainingEndDateInfo = ({ product }) => {
+  return (
+    <div className="card-list-item__row">
+      <div className="card-list-item-meta__items">
+        <div className="card-list-item-meta__item">
+          {product.user.training_ends_on ? (
+            <time className="a-meta" dateTime={product.user.training_ends_on}>
+              <span className="a-meta__label">研修終了日</span>
+              <span className="a-meta__value">
+                {product.user.training_ends_on}
+              </span>
+              {product.user.training_remaining_days === 0 ? (
+                <span className="a-meta__value is-danger">
+                  （本日研修最終日）
+                </span>
+              ) : product.user.training_remaining_days < 1 ? (
```

Comment:
こちらは元々、研修期間が終了しているかどうかの判断だったはずです。
なので、以下になるかと思います！
```suggestion
              ) : product.user.training_remaining_days < 0 ? ( #もしくはnetagive?
```

Reply:
@reckyy 
すみません、おかしなところを変更してしまっておりました😭
確認不足で申し訳ありません。

再度修正しておりますので、お手すきの際にご確認よろしくお願いいたします。